recipes/tamper/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin/
+mkdir -p ${PREFIX}/share/tamper/
+mkdir -p ${PREFIX}/share/tamper/src/
+mkdir -p ${PREFIX}/share/tamper/checkpoints/trained
+cp -r src/. ${PREFIX}/share/tamper/src/
+cp -r checkpoints/trained/. ${PREFIX}/share/tamper/checkpoints/trained
+ls ${PREFIX}/share/tamper/src/
+
+#This allows src code to be executbale for user and group
+chmod -R u+x ${PREFIX}/share/tamper/src/
+chmod -R g+x ${PREFIX}/share/tamper/src/
+
+ls -lah ${PREFIX}/share/tamper/src/
+
+#Adding python executable for the script to work
+echo "#!/bin/bash" > ${PREFIX}/bin/predict_tAMPer
+echo "python ${PREFIX}/share/tamper/src/predict_tAMPer.py \"\$@\"" >> ${PREFIX}/bin/predict_tAMPer
+
+echo "#!/bin/bash" > ${PREFIX}/bin/train_tAMPer.py
+echo "python ${PREFIX}/share/tamper/src/train_tAMPer.py \"\$@\"" >> ${PREFIX}/bin/train_tAMPer
```

Comment:
Why do you want to do this?
usually you use https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points for this use-case